mypy/build.py

```diff
@@ -413,12 +422,13 @@ def correct_rel_imp(imp: Union[ImportFrom, ImportAll]) -> str:
 
             return new_id
 
-        res = []  # type: List[Tuple[str, int]]
+        res = []  # type: List[Tuple[int, str, int]]
```

Comment:
What if the same module is present multiple times here with different priorities? Should we choose the highest priority?